src/zenml/integrations/deepchecks/__init__.py

```diff
@@ -0,0 +1,64 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Deepchecks integration for ZenML.
+
+The Deepchecks integration provides a way to validate your data in your pipelines.
+It includes a way to detect data anomalies and define checks to ensure quality of
+data.
+
+The integration includes custom materializers to store Deepchecks `SuiteResults` and
+a visualizer to visualize the results in an easy way on a notebook and in your
+browser.
+"""
+
+from typing import List
+
+from zenml.enums import StackComponentType
+from zenml.integrations.constants import DEEPCHECKS
+from zenml.integrations.integration import Integration
+from zenml.zen_stores.models.flavor_wrapper import FlavorWrapper
+
+DEEPCHECKS_DATA_VALIDATOR_FLAVOR = "deepchecks"
+
+
+class DeepchecksIntegration(Integration):
+    """Definition of [Deepchecks](https://github.com/deepchecks/deepchecks) integration for ZenML."""
+
+    NAME = DEEPCHECKS
+    REQUIREMENTS = ["deepchecks[vision]==0.8.0", "torchvision==0.11.2"]
```

Comment:
why does deepchecks integration require torchvision? I can understand why you've included the optional vision beta code, but less clear why torchvision.

Reply:
this may be an oversight in how Deepchecks is packaged, but torchvision is required for the vision support and not installed as a dependency. I had to include it explicitly

Reply:
In this case I am a bit concerned that our vision package is blowing this integration up. Why don't we have for now two seperate integrations like so: `deepchecks` and `deepchecks_vision`?

Reply:
@htahir1 interesting idea, but this could be tricky, they are like two sides of the same coin. Let me run some experiments to see how doable this is.